src/main/java/org/recommendation/data/helper/MovieDataHelper.java

```diff
@@ -0,0 +1,24 @@
+package org.recommendation.data.helper;
+
+import org.recommendation.model.Movie;
+import org.recommendation.model.User;
+import org.recommendation.service.Filter;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public class MovieDataHelper {
+    public static Map<String, Movie> movieMap;
```

Comment:
where are we initialising this ? and why it is public?

Reply:
initializing this iniside MovieDataInitializerImpl, it is public because its shared through out the system where we required movie details.